lib/dl_file_uploader_worker_lib/dl_file_uploader_worker_lib/utils/parsing_utils.py

```diff
@@ -13,7 +13,7 @@
 )
 
 import attr
-import cchardet as chardet
+import cchardet as chardet  # type: ignore[import-untyped]
```

Comment:
If an untyped module has no stubs, do this instead of ignoring every import
https://github.com/datalens-tech/datalens-backend/blob/1ebc641857bdd9be1dae4e28e3450d23ff5fd6d7/lib/dl_file_uploader_worker_lib/pyproject.toml#L68-L70
(I see that we already have cchardet there. Is something wrong with it?)

Reply:
My guess is this change is unnecessary https://github.com/datalens-tech/datalens-backend/pull/293/files#diff-dd66c4a68748aadce0343e7c6ca351c8f84c7814877fee1640cfa23b1bfe0ac2R69

Reply:
Seems that faust-cchardet override can be useless. Will make changes and check.

Reply:
Is this ignore still needed?

Reply:
Having this issue on local machine, maybe wiil be fixed for CI